addons/project/static/tests/project_task_kanban_renderer_tests.js

```diff
@@ -1,124 +0,0 @@
-/** @odoo-module */
```

Comment:
in stable, it would be better to keep that file and just fix the issue instead of moving the test in another file.

(in master, I am not against to move the test)